pandas/tests/arrays/boolean/test_astype.py

```diff
@@ -51,3 +51,18 @@ def test_astype_to_integer_array():
     result = arr.astype("Int64")
     expected = pd.array([1, 0, None], dtype="Int64")
     tm.assert_extension_array_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "func",
+    [
+        lambda s: s.tolist()[0],
+        lambda s: s.to_dict()[0],
+        lambda s: list(s.iteritems())[0][1],
+        lambda s: list(iter(s))[0],
+    ],
+)
+def test_conversion_methods_return_type_is_native(func):
```

Comment:
We could maybe move this test to the common arrays/masked/ section? We should be able to know the expected python class for each of the data types

Reply:
For sure. I'll make a separate file there (call it `test_conversions.py` maybe)